lib/HTFeed/PackageType/IA/ImageRemediate.pm

```diff
@@ -19,6 +19,9 @@ sub run {
     my $objid         = $volume->get_objid();
     my $scandata_xpc  = $volume->get_scandata_xpc();
     my $resolution    = $volume->get_db_resolution();
+    my $labels        = {packagetype => 'ia'};
+    my $start_time    = $self->{job_metrics}->time;
+    my $page_count    = $volume->get_page_count();
```

Comment:
Tests, e.g. `t/ia.t`, are currently failing because these `$volume->get_page_count()` say they have zero pages.

Reply:
If I move the `$volume->get_page_count()` to after the main body of work has been done I get the correct value.